src/scicat_offline_ingestor.py

```diff
@@ -124,3 +134,6 @@ def main() -> None:
                 scicat_origdatablock,
             )
             raise RuntimeError("Failed to create dataset or origdatablock.")
+
+if __name__ == "__main__":
+    main()
```

Comment:
Also these. You can just type ``background_ingestor`` instead of using the file itself as a script.